### DISCUSSION TITLE
How to save coc plugins list?

### DISCUSSION THREAD

**Discussion by andywu1998** [neoclide/coc.nvim#4753] [Q&A] [answered]
I uploaded my  ~/.config/nvim directory to github, so that when I use a new PC, I clone it, and then lazy(my plug manager) will install and configure plugins for me. 

So I want to know, can I save the plugins list of Coc as a directory or a file? so that I can easily sync my config to another PC.

**Comment by fannheyward** [comment, marked as answer]
Check `:h g:coc_global_extensions`, you can save extensions list to this, and coc.nvim will automatically install the missing/unfound extensions.

**Comment by chemzqm** [comment]
`:h coc-extensions-npm`

**Comment by andywu1998** [comment]
thank you both!
